test/TickMath.starknet.ts

```diff
@@ -0,0 +1,188 @@
+    })
+
+    for (const absTick of [
+      50,
+      100,
+      250,
+      500,
+      1_000,
+      2_500,
+      3_000,
+      4_000,
+      5_000,
+      50_000,
+      150_000,
+      250_000,
+      500_000,
+      738_203,
+    ]) {
+      for (const tick of [-absTick, absTick]) {
+        describe(`tick ${tick}`, () => {
+          it('is at most off by 1/100th of a bips', async () => {
+            const jsResult = new Decimal(1.0001).pow(tick).sqrt().mul(new Decimal(2).pow(96))
+            const result = await tickMath.getSqrtRatioAtTick_986cfba3(new BN(tick).toTwos(24).toString())
+            const absDiff = new Decimal(result.toString()).sub(jsResult).abs()
+            expect(absDiff.div(jsResult).toNumber()).to.be.lt(0.000001)
+          })
+          /*
+          it('result', async () => {
+            expect((await tickMath.getSqrtRatioAtTick_986cfba3(tick)).toString()).to.matchSnapshot()
+          })
+          
+          it('gas', async () => {
+            await snapshotGasCost(tickMath.getGasCostOfgetSqrtRatioAtTick_986cfba3(tick))
+          })*/
+        })
+      }
+    }
+  })
+
+  describe('#MIN_SQRT_RATIO', async () => {
+    it('equals #getSqrtRatioAtTick(MIN_TICK)', async () => {
+      const min = await tickMath.getSqrtRatioAtTick_986cfba3(new BN(MIN_TICK).toTwos(24).toString())
+      const result = await tickMath.MIN_SQRT_RATIO_ee8847ff()
+      expect(min[0].toNumber()).to.eq(result[0].toNumber())
+      expect(min[0].toNumber()).to.eq(MIN_SQRT_RATIO)
+    })
+  })
+
+  describe('#MAX_SQRT_RATIO', async () => {
+    it('equals #getSqrtRatioAtTick(MAX_TICK)', async () => {
+      const max = await tickMath.getSqrtRatioAtTick_986cfba3(new BN(MAX_TICK).toString())
+      const result = await tickMath.MAX_SQRT_RATIO_6d2cc304()
+      expect(max[0].toString()).to.eq(result[0].toString())
+      expect(max[0].toString()).to.eq(MAX_SQRT_RATIO.toString())
+    })
+  })
+
+  describe('#getTickAtSqrtRatio', () => {
+    /*
+    it('throws for too low', async () => {
+      expect(await tickMath.getTickAtSqrtRatio_4f76c058((MIN_SQRT_RATIO.sub(1)).toString())).to.be.revertedWith('R')
+    })
+
+    it('throws for too high', async () => {
+      expect(await tickMath.getTickAtSqrtRatio_4f76c058((MAX_SQRT_RATIO).toString())).to.be.revertedWith('R')
+    })
+    */
+    it('ratio of min tick', async () => {
+      const result = await tickMath.getTickAtSqrtRatio_4f76c058("4295128739")
+      expect(result[0].toString()).to.eq(new BN(MIN_TICK).toTwos(24).toString())
+    })
+    it('ratio of min tick + 1', async () => {
+      const result = await tickMath.getTickAtSqrtRatio_4f76c058('4295343490')
+      expect(result[0].toString()).to.eq(new BN(MIN_TICK + 1).toTwos(24).toString())
+    })
+    it('ratio of max tick - 1', async () => {
+      const result = await tickMath.getTickAtSqrtRatio_4f76c058('42914613736366300043187065181887844931066902546562495343490')
```

Comment:
From TickMath.spec.ts it seems like the parameter value should be `'1461373636630004318706518188784493106690254656249'`